lib/tokens/index.js

```diff
@@ -8,7 +8,7 @@ const error = require('../error')
 
 module.exports = (log, config) => {
   config = config || {}
-  const lifetimes = config.tokenLifetimes || {
+  const lifetimes = config.tokenLifetimes = config.tokenLifetimes || {
```

Comment:
Slight change to avoid updating loads of tests that use `SessionToken` without passing in `tokenLifetimes`.